test/extensions/filters/http/jwt_authn/extractor_test.cc

```diff
@@ -265,6 +274,30 @@ TEST_F(ExtractorTest, TestCustomParamToken) {
   tokens[0]->removeJwt(headers);
 }
 
+// Test extracting token from a cookie
+TEST_F(ExtractorTest, TestCookieToken) {
+  auto headers = TestRequestHeaderMapImpl{
+      {"cookie", "token-cookie=token-cookie-value; token-cookie-2=token-cookie-value-2"},
+      {"cookie", "token-cookie-3=token-cookie-value-3"}};
```

Comment:
Also a quoted cookie value could be added here.

Reply:
Done!